_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -28,11 +28,11 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any `iframe` element that has a negative number as a `tabindex` [attribute value][].
+This rule applies to any non-focusable `iframe` element that has focusable content.
 
 ## Expectation
 
-For each test target, the [nested browsing context][] does not include elements that are [visible][] and part of the [sequential focus navigation][]. An element is "included" in a [nested browsing context][] if its [owner document][] is the [container document][] of the [nested browsing context][].
+For each test target, the [nested browsing context][] does not contain elements that are [visible][] and part of the [sequential focus navigation][]. 
```

Comment:
Getting into technical nerd mode here also 🤓 
A [browsing context container](https://html.spec.whatwg.org/multipage/browsers.html#browsing-context-container) (i.e. `iframe`) "instantiate" a "browsing context" which is called "nested" because the `iframe` itself is already inside another browsing context (typically the browser window).

Now, the (nested) [browsing context](https://html.spec.whatwg.org/multipage/browsers.html#browsing-context) only has a [container document](https://html.spec.whatwg.org/multipage/browsers.html#bc-container-document). In turn, this document contains elements (or rather "own" them). 

As much as I agree that the simplified text proposed here makes sense at the human level, it is not technically correct and I'm not sure it is sufficiently unambiguous to be used in an ACT rule (first draft of the rule only said "include" without the precision, but that was deemed unclear at the time: https://github.com/act-rules/act-rules.github.io/pull/1515#discussion_r542392991)

I have no problem replacing "include" by "contain", but I'd rather keep the precision in some form or other. To make the Expectation look lighter, we could move the second sentence to a new line/paragraph (possibly with a link after #1744  has landed):
```markdown
For each test target, the [nested browsing context][] does not [contain](#akn7bn:contain) elements that are [visible][] and part of the [sequential focus navigation][].

An element is <dfn id="#akn7bn:contain">contained</dfn> in a [nested browsing context][] if its [owner document][] is the [container document][] of the [nested browsing context][].
```
or fully move it to a new definition.